src/components/Book.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBookFetch } from '../redux/books/books';
+import classes from './Book.module.css';
+
+const Book = (props) => {
+  const { id, book } = props;
+  const { title, author, category } = book;
+  const dispatch = useDispatch();
+
+  const removeBook = () => {
+    dispatch(removeBookFetch(id));
+  };
+
+  return (
+    <li className={classes.books}>
+      <div className={classes.book__list}>
+        <h3 className={classes.book__categories}>{category}</h3>
+        <h2>{title}</h2>
+        <h3 className={classes.book__author}>{author}</h3>
+      </div>
+      <button type="button" className={classes.comment__btns}>
+        Comments
+      </button>
+      <button
+        type="button"
+        onClick={removeBook}
+        className={classes.remove__btns}
+      >
+        Remove
+      </button>
+      <button type="button" className={classes.edit__btns}>
+        Edit
+      </button>
+    </li>
+  );
```

Comment:
- [ ] Please work out the font typefaces difference that is in this part of the Book component. The book category should have the same as the buttons below it. Also the should be aligned  on same margin 

your design
![image](https://user-images.githubusercontent.com/90222110/225727385-d0c680b4-7e38-497e-b5d7-246a0ff38bf0.png)

Given design
![image](https://user-images.githubusercontent.com/90222110/225727544-6c3e1157-8cb1-4852-aeb5-2638200cfb34.png)